### PR TITLE
u3d/install: allow to download from one platform while on another one

### PR DESCRIPTION
**Pull request by lacostej**

### CODE REVIEW

File: lib/u3d/commands.rb [hunk 1]
```diff
@@ -103,7 +103,8 @@ def list_available(options: {})
       def install(args: [], options: {})
         version = specified_or_current_project_version(args[0])
 
-        os = U3dCore::Helper.operating_system
+        UI.user_error!("install can only work with current operating system. You specified #{options[:operating_system]}") if options[:install] && options[:operating_system]
```
Comment (niezbop) [inline lib/u3d/commands.rb]: I would maybe specify the use of the --no-install here

File: lib/u3d/commands_generator.rb [hunk 2]
```diff
@@ -139,10 +140,12 @@ def run
         c.option '--[no-]download', 'Perform or not downloading before installation. Downloads by default'
         c.option '--[no-]install', 'Perform or not installation after downloading. Installs by default'
         c.option '-p', '--packages PACKAGES', Array, 'Specifies which packages to download/install. Overriden by --all'
+        c.option '-o', '--operating_system STRING', String, "Downloads packages for specific OS [#{oses.join(', ')}]"
```
Comment (niezbop) [inline lib/u3d/commands_generator.rb]: We could say here as well that this is a precision for --no-install. Something like `Requires the --no-install option`

**Comment by niezbop** [comment]
I think we should be a bit clearer regarding this option, and clearly state that this is a focus for `--no-install`. Great addition besides that.

File: lib/u3d/commands.rb (hunk 1 above)
Comment (lacostej) [reply lib/u3d/commands.rb]: At this point I don't know if the user specified install explicitly or if it was the default. 

Is this better?
```
UI.user_error!("You cannot use the --operating_system and the --install options together") if options[:install] && options[:operating_system]
```

Comment (niezbop) [reply lib/u3d/commands.rb]: Much better indeed!

**Comment by lacostej** [comment]
Simple change that allowed me to implement part of the #227 from my Mac :)

**Pull request merged**